### PR TITLE
Main readme changes + moving away module specific stuff

### PR DESCRIPTION
**Pull request by glibsm**
I feel like readmes about the modules do not belong in the main wiki and are much better placed in their respective sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,14 +60,12 @@ func main() {
...
 
-It's common for a service to handle many different workloads.  For example, a
-service may expose RPC endpoints and also ingest Kafka messages.  In the Python
-world, this means different deployments that run with different entry points.
-Due to Python's threading model, this was required.
+It's common for a service to handle many different workloads. For example, a
+service may expose RPC endpoints and also ingest Kafka messages.
 
-In the Go service, we can have a simpler model where we create a single binary,
+In uberfx, there is a simpler model where we create a single binary,
```
Comment (sectioneight) [inline README.md]: UberFx

File: README.md [hunk 2]
```diff
@@ -1,36 +1,30 @@
-# UberFx Service Framework
+# Uber FX Service Framework
```
Comment (sectioneight) [inline README.md]: UberFx

File: modules/rpc/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# RPC Module
```
Comment (sectioneight) [inline modules/rpc/README.md]: This should probably be godoc-style so it shows up on godoc.org under the package

File: modules/uhttp/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# HTTP Module
```
Comment (sectioneight) [inline modules/uhttp/README.md]: godoc

File: modules/rpc/README.md (hunk 3 above)
Comment (glibsm) [inline modules/rpc/README.md]: are you saying that there is a way to get there readmes to work with go-doc or that we should replace these reamdes with godoc?

**Pull request merged**